sample/src/androidTest/java/com/schibsted/spain/barista/sample/AssertionsTest.java

```diff
@@ -271,12 +271,22 @@ public void checkBackButton() {
   }
 
   @Test
-  public void checkSameDrawable() throws Exception {
+  public void checkDrawable_withId_withSameDrawable() throws Exception {
```

Comment:
Test names look confusing, it had to close my eyes really hard and think with all the power of my brain to understand what it means.

Here's a debatable proposal:
- checkDrawable_withId
- checkDrawable_withId_failure
- checkDrawable_withAnyDrawable
- checkDrawable_withAnyDrawable_failure

Reply:
Mmmm... it's not the moment to debate 😂 let's finish all this crap